src/bidiMapper/domains/context/browsingContextImpl.ts

```diff
@@ -200,7 +208,8 @@ export class BrowsingContextImpl {
 
   #updateConnection(cdpClient: CdpClient, cdpSessionId: string) {
     if (!this.#targetDefers.targetUnblocked.isFinished) {
-      this.#targetDefers.targetUnblocked.reject('OOPiF');
+      this.#targetDefers.targetUnblocked = new Deferred<void>();
+      log('OOPiF');
     }
     this.#targetDefers.targetUnblocked = new Deferred<void>();
```

Comment:
In this version, if code is waiting for `await this.#targetDefers.targetUnblocked`, it will be never resolved in case of OOPIF, as long as the `this.#targetDefers.targetUnblocked` will be assigned to another deferred. I suggest the following behaviour:

```suggestion
      // No need it re-creating the deferred.
      log('targetUnblocked postponed because of OOPiF');
    } else {
      this.#targetDefers.targetUnblocked = new Deferred<void>();
    }
```